### ISSUE TITLE
Repository License?

### ISSUE THREAD

**Issue by davefollett** [cassidoo/blahg#20]
I've been using your repository as inspiration but I just noticed that it does not include a license.  So technically (according to the google machine):

> If a repository lacks a license, it means that the code within it is considered under the default copyright laws, essentially granting no permission to anyone else to use, modify, or distribute the code without explicit consent from the creator; in simpler terms, nobody can use the code unless they get permission directly from the copyright holder.

I _think_ this is on accident but I want to make sure I'm not doing something I shouldn't.  Thanks.

**Comment by cassidoo** [comment]
Ooh, you can just use this! I'll add a license so there's no confusion 👍🏼

**Comment by justaugustus** [comment]
Thanks Cassidy!!
